src/constants/serverUri.js

```diff
@@ -0,0 +1,3 @@
+
+export const ServerHostName = 'https://localhost:5001';
+export const ServerHostName_2 = 'http://localhost:5000';
```

Comment:
I think... We can ignore the `http://localhost:5000` since (I think) it's just the backend's react-app

Reply:
True, but the extra unused constant does not break the app, so why not :D